bugwarrior/docs/services/jira.rst

```diff
@@ -68,18 +68,22 @@ If your Jira instance is only available over HTTPS, and you're running into
 
     jira.verify_ssl = False
 
-Import Labels as Tags
-+++++++++++++++++++++
+Import Labels and Sprints as Tags
++++++++++++++++++++++++++++++++++
 
 The Jira issue tracker allows you to attach labels to issues; to
 use those labels as tags, you can use the ``jira.import_labels_as_tags``
 option::
 
     jira.import_labels_as_tags = True
 
-Also, if you would like to control how these labels are created, you can
-specify a template used for converting the Jira label into a Taskwarrior
-tag.
+You can also import the names of any sprints associated with an issue as tags,
+by setting the ``jira.import_sprints`` option::
+
+    jira.import_sprints = True
```

Comment:
I know you're going to hate me for suggesting this, but even though I might regret naming `import_labels_as_tags` what I did name it, it might be less confusing for future readers if both <thing-to-tag> import configuration options were named similarly.